Source/JavaScriptCore/b3/B3Common.cpp

```diff
@@ -70,10 +70,10 @@ bool shouldSaveIRBeforePhase()
     return Options::verboseValidationFailure();
 }
 
-std::optional<GPRReg> pinnedExtendedOffsetAddrRegister()
+std::optional<GPRReg> linkRegister()
```

Comment:
Do we want some kind of RAII object to make sure two places don't do this with overlapping live-ranges?